.github/workflows/push_trigger.yml

```diff
@@ -19,10 +19,6 @@ jobs:
       - name: Install npm dependencies
         run: |
           npm install
-      - name: Updating serverBaseURL.
-        run: |
-          find . -type f -name "*build.gradle" -print0 | xargs -0 sed -i "s/api-internal.sandbox.xyz.net/${{ github.event.inputs.defaultServerBaseURL }}/g"
```

Comment:
Why is this block removed?

Reply:
I was not aware, it was a conflict emerged upon rebase. I will put it back.

Reply:
can you revert this file as well

Reply:
![image](https://github.com/mosip/android-registration-client/assets/56625754/5c41de11-bd45-4838-b05b-04756d2609fe)